thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -88,14 +112,18 @@ export default Controller.extend({
     function() {
       const severityMap = this.get('severityMap');
       const patternMap = this.get('patternMap');
+      const amountChange = this.get('customPercentChange');
       const selectedPattern = this.get('selectedTunePattern');
       const selectedSeverity = this.get('selectedSeverityOption');
+      const isPercent = selectedSeverity === 'Percentage of Change';
       const mttdVal = Number(this.get('customMttdChange')).toFixed(2);
-      const severityThresholdVal = (Number(this.get('customPercentChange'))/100).toFixed(2);
+      const severityThresholdVal = isPercent ? (Number(amountChange)/100).toFixed(2) : amountChange;
```

Comment:
fyi, if this logic is only used in the template, we do have template helpers such as `format-number` for UI.

Reply:
good reminder. This is actually used as a param sent to the model method `triggerTuningSequence`